db/migrate/20160124145702_create_comments.rb

```diff
@@ -0,0 +1,12 @@
+class CreateComments < ActiveRecord::Migration
+  def change
+    create_table :comments do |t|
+      t.integer :user_id
+      t.string :content
+      t.integer :votes
+      t.references :comment_on, polymorphic: true, index: true
```

Comment:
Good observation. I just felt in normal grammar. "This is a `comment_on` answer or question" seems more readable than "This comment is `commentable` on answer or question".

What do you think. Should we still go with `commentable` or leave it as it is?